mission_list.md

```diff
@@ -0,0 +1,69 @@
+# Mission List
+
+This list states the projects that PoToC maintains, as required by the [constitution](https://github.com/polkadot-tooling-collective/constitution):
+
+(this is a draft for anyone to comment on and challenge the decisions)
```

Comment:
Instructions like, "to add or remove projects, {make a PR, submit an RFC, something else}". I guess once merged it's no longer a draft and will be part of the proposal?

Reply:
Yes instructions will be added, but i will wait until we have a process to not be flooded with mediocre applications.